telethon/client/messages.py

```diff
@@ -826,6 +826,8 @@ async def callback(event):
                 await client.send_message(chat, 'Hi, future!', schedule=timedelta(minutes=5))
         """
         if file is not None:
+            if isinstance(message, types.Message):
+                message = message.message
```

Comment:
`send_message` of a `Message` without file copies the formatting entities, but they're lost here.

Reply:
What do you mean?

Reply:
`message.message` is the raw, unformatted text. We need to pass the ´formatting_entities` argument below, or all formatting will be lost. But, probably only if the user did not specify their *own* formatting entities as input.

Reply:
Fair enough

Reply:
Done